templates/pod/acp/.acr.hbs

```diff
@@ -0,0 +1,40 @@
+# Root ACR for the agent account
+@prefix acl: <http://www.w3.org/ns/auth/acl#>.
+@prefix acp: <http://www.w3.org/ns/solid/acp#>.
+
+# The owner has full access to every resource in their pod.
+# Other agents have no access rights,
+# unless specifically authorized in other ACRs.
+<#root>
+    a acp:AccessControlResource;
+    # Set the access to the root storage folder itself
+    acp:resource <./>;
+    # The homepage is readable by the public
+    acp:accessControl <#fullOwnerAccess>, <#publicReadAccess>;
+    # All resources will inherit this authorization
+    acp:memberAccessControl <#fullOwnerAccess>.
+
+# The public only has read access
+<#publicReadAccess>
+    a acp:AccessControl;
+    acp:apply [
+        a acp:Policy;
+        acp:allow acl:Read;
+        acp:anyOf [
+            a acp:Matcher;
+            acp:agent acp:PublicAgent
+        ]
+    ].
+
+# The owner has all of the access modes allowed
+<#fullOwnerAccess>
+    a acp:AccessControl;
+    acp:apply [
+        a acp:Policy;
+        acp:allow acl:Read, acl:Write, acl:Control;
+        acp:allOf [
+            a acp:Matcher;
+            acp:agent <{{webId}}>
+            acp:client <https://solid.pondersource.com/clientid.jsonld>
```

Comment:
Sorry for hard-coding this, but I can't see how to add a handlebars template var in CSS, and leaving it out would mean any app on the web can get full owner access.

Reply:
committing this as-is now, since it's better than nothing.